start-plugin.sh

```diff
@@ -4,5 +4,5 @@ set -e
 
 docker run --name=weaveplugin --privileged -d \
     --net=host -v /var/run/docker.sock:/var/run/docker.sock \
-    -v /usr/share/docker/plugins:/usr/share/docker/plugins \
+    -v /run/docker/plugins:/usr/share/docker/plugins \
```

Comment:
Should the target `/usr/share/docker/plugins` of the volume mount match the default value `/run/docker/plugins/weave.sock` for the socket from above? (I know I did not have this lined up previously)